controllers/storagecluster/cephblockpools.go

```diff
@@ -69,6 +71,22 @@ func (r *StorageClusterReconciler) newCephBlockPoolInstances(initData *ocsv1.Sto
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      builtinMgrPoolName,
+				Namespace: poolNamespace,
+			},
+			Spec: cephv1.NamedBlockPoolSpec{
+				Name: ".mgr",
+				PoolSpec: cephv1.PoolSpec{
+					DeviceClass:    initData.Status.DefaultCephDeviceClass,
+					FailureDomain:  getFailureDomain(initData),
+					Replicated:     generateCephReplicatedSpec(initData, "data"),
+					EnableRBDStats: true,
```

Comment:
no need for rbd stats enabled on this pool
```suggestion
					EnableRBDStats: false,
```